### PR TITLE
Quality: PowerShell Summit GH-3373

### LINKED ISSUE #3373: Quality: Install-AdcsOnlineResponder
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Formatting code samples, Formatting command syntax

### Article List

- https://github.com/MicrosoftDocs/windows-powershell-docs/blob/main/docset/winserver2022-ps/adcsdeployment/Install-AdcsOnlineResponder.md

### PR DESCRIPTION
**Pull request by Snozzberries**
# PR Summary

- Resolves #3373

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsOnlineResponder.md [hunk 1]
```diff
@@ -124,12 +138,13 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ### Microsoft.CertificateServices.Deployment.Common.OCSP.OnlineResponderResult
 
 ## NOTES
-* Ensure you run Windows PowerShell as an administrator. You can use the *Force* parameter to bypass the prompt for confirmation.
-To see parameters, run the following command: `Install-AdcsOnlineResponder -?`
+
+- Ensure you run Windows PowerShell as an administrator. You can use the **Force** parameter to
+  bypass the prompt for confirmation. To see parameters, run the following command:
+  `Install-AdcsOnlineResponder -?`
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsOnlineResponder.md:144]:
```suggestion

    `Install-AdcsOnlineResponder -?`
```

Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsOnlineResponder.md:144]:
```suggestion

    `Install-AdcsOnlineResponder -?`
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Snozzberries, minor suggested change, please.

**Comment by Snozzberries** [comment]
Commit applied. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**